<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>615</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Import data</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="SELFILE_LBL">
        <property name="text">
         <string>Load from</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="SELFILE_TXT"/>
      </item>
      <item>
       <widget class="QToolButton" name="SELFILE_BTN">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_5" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QComboBox" name="VAR_CB"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="VAR_LBL">
        <property name="text">
         <string>Variable</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_6" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QComboBox" name="SENSOR_CB"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="SENSOR_LBL">
        <property name="text">
         <string>Measuring point</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="TC_LBL">
        <property name="text">
         <string>Time column</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="TC_SB">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="VC_LBL">
        <property name="text">
         <string>Value Column</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="VC_SB">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="FORMAT_LBL">
        <property name="text">
         <string>time format</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="FORMAT_TXT">
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>%Y/%m/%d</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>%d/%m/%Y</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>%d-%m-%Y</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>%Y/%m/%d %H:%M</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="SKIPLINES_LBL">
        <property name="text">
         <string>Skip lines</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="SKIPLINES_SB">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="SEP_LBL">
        <property name="text">
         <string>Separator</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="SEP_TXT">
        <property name="text">
         <string>,</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Table preview</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="TABLE_PREVIEW">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <column/>
     <column/>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_7" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="OVERWRITE_CB">
        <property name="text">
         <string>Overwrite existing data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="SAVEEDIT_CB">
        <property name="text">
         <string>Save edits</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="MSG_LBL">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
